.travis.yml

```diff
@@ -3,9 +3,6 @@ python:
   - '2.7'
 
 before_install:
-  - eval export AWS_ACCESS_KEY_ID=\$${TRAVIS_BRANCH}_AWS_ACCESS_KEY_ID
```

Comment:
These are now set without a branch prefix in Travis's settings, thus no need to set them here.